### PR TITLE
Suppress warning about Is.StringContaining on NUnit 3

### PR DESCRIPTION
**Pull request by RoosterDragon**
A side effect of moving to NUnit 3, I believe.

### CODE REVIEW

**Comment by RoosterDragon** [comment]
Hmm, is Travis still running 2?

**Comment by pchote** [comment]
Preference is still for 2 if there is a system install.  It will only use nunit 3 if you fetch a local copy.

**Comment by atlimit8** [comment]
Actually, it tries in this order:
 1. NUnit 3 from NuGet
 2. System's `nunit3-console`
 3. System's `nunit2-console`
 4. System's `nunit-console`
--
So, NUnit 2 is still supported.

**Comment by RoosterDragon** [comment]
Well if we need to be able to run on 2, I can't switch to these methods since they don't exist!

Should I just disable the warnings in this section instead?

**Comment by RoosterDragon** [comment]
Changed to suppress the warnings.

File: OpenRA.Test/OpenRA.Game/ActorInfoTest.cs [hunk 1]
```diff
@@ -64,11 +64,14 @@ public void TraitsInConstructOrderB()
 			}
 			catch (Exception e)
 			{
+				// Is.StringContaining is deprecated in NUnit3, but we need to support NUnit2 so we ignore the warning.
```
Comment (atlimit8) [inline OpenRA.Test/OpenRA.Game/ActorInfoTest.cs]: NUnit 3
NUnit 2

**Comment by Mailaender** [comment]
:+1:

File: OpenRA.Test/OpenRA.Game/ActorInfoTest.cs (hunk 1 above)
Comment (Phrohdoh) [inline OpenRA.Test/OpenRA.Game/ActorInfoTest.cs]: What are you suggesting or pointing out here?

**Pull request merged**